bindings/java/java/src/main/java/org/ethereum/evmc/EvmcVm.java

```diff
@@ -156,14 +143,15 @@ public int set_option(String name, String value) {
   native int get_result_size();
 
   /**
-   * This method cleans up resources
-   *
-   * @throws Exception
+   * Utility method to get a bytebuffer address
+   * @param buffer the byte buffer to consider
+   * @return the byte buffer address
    */
+  native long address(ByteBuffer buffer);
```

Comment:
Adds a method to get the native address of a direct bytebuffer. It's not exposed in Java.